src/assets/cipher-card.css

```
@@ -49,18 +49,18 @@
   color: var(--cryptotron-text-secondary);
 }
 
-.theory-content :deep(h3) {
+.theory-content h3 {
   color: var(--neon-green);
   font-family: 'Orbitron', monospace;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Orbitron" (font-family-name-quotes)

(font-family-name-quotes)